src/main/java/homework3/ex1/page/Ex1LeftNavigationBar.java

```diff
@@ -0,0 +1,66 @@
+package homework3.ex1.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class Ex1LeftNavigationBar extends BasePage {
+
+    public Ex1LeftNavigationBar(WebDriver driver) {
```

Comment:
According to Java code convention methods must be lower than fields. Plus should be order:

fields 
constructor (if present)
methods

Fix pls